readme.md

```diff
@@ -1,7 +1,7 @@
-[![Build Status](https://travis-ci.org/mightyiam/eslint-config-standard-with-typescript.svg?branch=master)](https://travis-ci.org/mightyiam/eslint-config-standard-with-typescript)
+[![Build Status](https://travis-ci.org/standard/eslint-config-standard-with-typescript.svg?branch=master)](https://travis-ci.org/standard/eslint-config-standard-with-typescript)
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
-[![Greenkeeper badge](https://badges.greenkeeper.io/mightyiam/eslint-config-standard-with-typescript.svg)](https://greenkeeper.io/)
-[![Releases](https://coderelease.io/badge/mightyiam/eslint-config-standard-with-typescript)](https://coderelease.io/github/repository/mightyiam/eslint-config-standard-with-typescript)
+[![Greenkeeper badge](https://badges.greenkeeper.io/standard/eslint-config-standard-with-typescript.svg)](https://greenkeeper.io/)
```

Comment:
I've requested installation of Greenkeeper from the admin of this org. Perhaps installation this image would load?

Reply:
We already have Greenkeeper enabled on all repos in the org. It's strange that you needed to send a request. I logged into Greenkeeper and confirmed that they were just waiting for this PR to be merged: https://github.com/standard/eslint-config-standard-with-typescript/pull/91 which I have already done :)